runtime/fiber.c

```diff
@@ -60,6 +60,8 @@ void caml_change_max_stack_size (uintnat new_max_size)
   caml_max_stack_size = new_max_size;
 }
 
+#define NUM_STACK_SIZE_CLASSES 5
+
```

Comment:
This define isn't used (yet), is it ?

Reply:
Not sure I follow, I see `NUM_STACK_SIZE_CLASSES` used in the functions `caml_alloc_stack_cache` and `stack_cache_bucket` in `fiber.c`.